.github/workflows/deploy.yml

```diff
@@ -93,59 +100,56 @@ jobs:
           npm ci
           npm run build
+      - name: Set up Python ${{ env.PY_VERSION }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ env.PY_VERSION }}
       - name: Install Poetry
         uses: abatilo/actions-poetry@v2
         with:
           poetry-version: ${{ env.POETRY_VERSION }}
-      - name: Add requirement.txt
+          poetry-plugins: "poetry-plugin-export"
+      - name: Add requirements.txt
         run: |
           poetry export -f requirements.txt --output requirements.txt --without-hashes
           echo "Generated requirements.txt:"
           cat requirements.txt
-      - name: deploy DHL
+      - name: Deploy admin app
         uses: cloud-gov/cg-cli-tools@main
         with:
-          command: cf push --vars-file vars.prod.yml --strategy rolling --no-wait
+          command: cf push --vars-file vars.prod.yml --strategy rolling
           cf_org: gsa-datagov
-          cf_space: ${{vars.ENVIRONMENT_NAME}}
+          cf_space: prod
           cf_username: ${{secrets.CF_SERVICE_USER}}
           cf_password: ${{secrets.CF_SERVICE_AUTH}}
-      # to-do
-      # - name: smoke test
-      #   uses: cloud-gov/cg-cli-tools@main
-      #   with:
-      #     command: cf run-task harvesting-logic -c "/home/vcap/app/scripts/smoke-test.py" --name smoke-test
-      #     cf_org: gsa-datagov
-      #     cf_space: ${{vars.ENVIRONMENT_NAME}}
-      #     cf_username: ${{secrets.CF_SERVICE_USER}}
-      #     cf_password: ${{secrets.CF_SERVICE_AUTH}}
-      # - name: monitor task output
-      #   uses: cloud-gov/cg-cli-tools@main
-      #   with:
-      #     command: >
-      #       scripts/monitor-cf-logs.sh harvesting-logic smoke-test
-      #     cf_org: gsa-datagov
-      #     cf_space: ${{vars.ENVIRONMENT_NAME}}
-      #     cf_username: ${{secrets.CF_SERVICE_USER}}
-      #     cf_password: ${{secrets.CF_SERVICE_AUTH}}
-      - name: Create Issue if it fails 😢
-        if: ${{ failure() && github.ref == 'refs/heads/main' }}
-        uses: JasonEtco/create-an-issue@v2
-        env:
-          GITHUB_TOKEN: ${{ secrets.ADD_TO_PROJECT_PAT }}
-          GITHUB_JOB: ${{ toJson(github)['job'] }}
-          GITHUB_ATTEMPTS: ${{ github.run_attempt }}
-          LAST_COMMIT: ${{ github.sha }}
-          LAST_RUN_BY: ${{ github.actor }}
-          RUN_ID: ${{ github.run_id }}
-          REPO: ${{ github.repository }}
+
+  network-policies:
+    name: Add network-policies
+    needs:
+      - deploy-staging
+      - deploy-prod
+    runs-on: ubuntu-latest
+    environment: ${{ matrix.environ }}
+    strategy:
+      fail-fast: false
+      matrix:
+        command: [
+          "cf add-network-policy datagov-harvest mdtranslator --protocol tcp --port 8080"
+        ]
+        environ: ["staging", "prod"]
+    steps:
+      - name: proxy to web
+        uses: cloud-gov/cg-cli-tools@main
         with:
-          filename: .github/deploy_failure.md
-          update_existing: true
```

Comment:
What's the intention behind removing this feature? We have used this in the past to be notified and automatically create issues for triage...

Reply:
I'll put it back, not sure why someone took it out.